storage/2023-11-03/blob/containers/create.go

```diff
@@ -41,6 +44,30 @@ func (c Client) Create(ctx context.Context, containerName string, input CreateIn
 		return
 	}
 
+	// Retry the container creation if a conflicting container is still in the process of being deleted
+	retryFunc := func(resp *http.Response, _ *odata.OData) (bool, error) {
+		if resp != nil {
+			if resp.StatusCode == http.StatusConflict {
```

Comment:
We should be able to make this:

```suggestion
			if response.WasStatusCode(resp, http.StatusConflict) {
```